src/dmd/backend/cod3.d

```diff
@@ -1283,6 +1277,13 @@ version (MARS)
             }
             break;
 
+        case BCexit:
+            retregs = 0;
+            gencodelem(cdb,e,&retregs,true);
```

Comment:
I assume `retregs` can be never be `mST0` or `mST01` here. Is that right?

Reply:
Right, because of line 1281.